src/main/java/io/orangebeard/listener/v3client/datatype/Attribute.java

```diff
@@ -0,0 +1,22 @@
+package io.orangebeard.listener.v3client.datatype;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import javax.validation.constraints.NotNull;
```

Comment:
I do not see why we would want to add javax validation annotations to these entities?

Reply:
Copied all the required entities from listenerapi